app/exports/candidate_email_send_counts_export.yml

```diff
@@ -17,3 +17,13 @@ custom_columns:
   unique_recipients:
     type: integer
     description: The number of unique recipients of this email
+
+  mailer:
+    type: string
+    description: The number of unique recipients of this email
```

Comment:
```suggestion
    description: The Rails mailer which sent the email
```

Reply:
Yep, I missed that. Cheers @duncanjbrown!